apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveService.kt

```diff
@@ -515,10 +515,15 @@ class OppgaveService(
         saksbehandler: BrukerTokenInfo,
     ): OppgaveIntern {
         try {
+            // TODO: Må undersøke konsekvensen av å legge til Status.NY på [hentOppgaveUnderBehandling]
```

Comment:
Hva mener du her?

Reply:
`hentOppgaveUnderBehandling` henter alle behandlinger hvor denne er true:

`fun erUnderBehandling(): Boolean = this in listOf(UNDER_BEHANDLING, PAA_VENT, ATTESTERING, UNDERKJENT)`

Siden denne _ikke_ inneholder `Status.NY` medfører det en del krøll i tilfeller hvor en behandling er opprettet, men ikke tildelt.